fhir_transformer/src/transform/transformer.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from fhir_transformer.src.config.logger import create_logger
+from fhir_transformer.src.transform.fhir import create_instance
+from fhir_transformer.src.helper import get_topic_name
+
+logger = create_logger("transformer")
+
+
+class Transformer:
+    def __init__(self, producer):
+        self.producer = producer
+
+    def transform(self, row, analysis):
+        return create_instance(row, analysis)
```

Comment:
Sorry but the `instance` naming is a bit confusing I think, the output being a dictionary. I was expecting something like a server or a more complex object ahah. 
I am probably missing something here, maybe some FHIR vocabulary, but just a side note ;)

Reply:
No, you're right! `instance` is a bit overloaded as a word. What it means here is a single fhir object or a document in mongo.
Do you have an idea for a better naming?

Reply:
`fhir_object` or `fhir_document` sound good :)